Makefile

```diff
@@ -37,4 +48,9 @@ vet:
 
 .PHONY: test-e2e
 test-e2e:
-	go test -v ./test/e2e/ 
+	go test -v ./test/e2e/
+
+.PHONY: clean
+clean:
+	go clean
+	rm -f ./preflight*
```

Comment:
This seems risky.... Can we iterate over the them like the build_multi_arch target does?